### PR TITLE
Medical Doctor guidebook.

### PR DESCRIPTION
**Pull request by Owai-Seek**
## About the PR
Adds a guide for medical doctor, there's an alternative PR right now of a similar nature, but it's been sitting for about two weeks, and I figured I'd take a crack at it.

Aha, found it. This would be an alternative to: #15432

The errant Chemistry page in the first few screenshots is fixed, I just didn't want to re-take them. (I was originally going to do all of it at once, but just Medical Doctor by itself was rough.)

**Media**

image
image
image
image
image
image
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Adds Medical Doctor, Cryo, and Cloning Guidebooks

### CODE REVIEW

**Comment by QuietlyWhisper** [comment]
you forgot about cauterization

**Comment by OttoMaticode** [comment]
isnt the cloning system getting sacked soon?

**Review by QuietlyWhisper — APPROVED** [review]

**Comment by OctoRocket** [comment]
I thought that cloning was just moved off to science. It is still pretty important knowledge to know, so I would keep it.

**Comment by EmoGarbage404** [comment]
at a glance, all the info looks fine. Consider splitting Cryogenics and Cloning into separate articles, considering how long the single page is. It'd also make them easier to find.

**Comment by EmoGarbage404** [comment]
also use GuideHelp components for cloner, biomass reclaimer, scanner, cryopod, and other medical items in the guide.

File: Resources/ServerInfo/Guidebook/Medical/Medical Doctor.xml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+- gas pump: Limits the network pressure, so those in the tube aren't crushed.
+- freezer: Chills the air, and by extention, patients in the tube, so Cryoxadone can activate.
+- gas filters: Removes waste gas (Carbon Dioxide and Nitrous Oxide) from the loop.
+- air out: Waste gasses go here, either into a storage tank, or into the waste network.
+<Box>
+<GuideEntityEmbed Entity="GasPipeBend" Rotation="90"/>
+<GuideEntityEmbed Entity="GasPipeStraight" Rotation="90"/>
+<GuideEntityEmbed Entity="GasPipeStraight" Rotation="90"/>
+<GuideEntityEmbed Entity="GasPipeBend" Rotation="0"/>
+<GuideEntityEmbed Entity="CloningPod" Caption="cryo pod" Rotation="0"/>
```
Comment (20kdc) [inline Resources/ServerInfo/Guidebook/Medical/Medical Doctor.xml:107]: Uhhh... is this intended to be correct? There's a dedicated entity prototype for cryo pods, `CryoPod`.

Comment (deltanedas) [reply Resources/ServerInfo/Guidebook/Medical/Medical Doctor.xml]: why is cryo pod even bundled in with cloning its unrelated

File: Resources/ServerInfo/Guidebook/Medical/Cryogenics.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<Document>
+# Cryogenics
+Cryogenics can be a bit daunting to understand, but hopefully, quick run through of this diagram, you'll learn how to work it! So, let's go over what each part does:
+
+- air in: Air comes in here through the distro or an air tank, giving those inside something to breathe, and the freezer something to chill.
+- gas pump: Limits the network pressure, so those in the tube aren't crushed.
+- freezer: Chills the air, and by extension, patients in the tube, so Cryoxadone can activate.
+- gas filters: Removes waste gas (Carbon Dioxide and Nitrous Oxide) from the loop.
+- air out: Waste gasses go here, either into a storage tank, or into the waste network.
```
Comment (moonheart08) [inline Resources/ServerInfo/Guidebook/Medical/Cryogenics.xml:9]:
```suggestion
- Air in: Air comes in here through the distro or an air tank, giving those inside something to breathe, and the freezer something to chill.
- Gas pump: Limits the network pressure, so those in the tube aren't crushed.
- Freezer: Chills the air, and by extension, patients in the tube, so Cryoxadone can activate.
- Gas filters: Removes waste gas (Carbon Dioxide and Nitrous Oxide) from the loop.
- Air out: Waste gasses go here, either into a storage tank, or into the waste network.
```

**Pull request merged**